### PR TITLE
yeah I know the resolution is hard-coded, get it from the dashing-kee…

### PR DESCRIPTION
**Pull request by jameswhite**
…palive file var

### CODE REVIEW

File: modules/autologinpi/boot_into_only_chrome [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+cat<<EOF > ~/.bash_profile
+if [ -z "$DISPLAY" ] && [ $(tty) == /dev/tty1 ]; then
+    /usr/bin/setterm -blank 0 -powerdown off
+    /usr/bin/startx
+    logout
+fi
+
+cat<< EOF > /home/pi/.xinitrc
+/usr/bin/unclutter &
+/usr/bin/chromium --window-size=1824,984 --incognito --kiosk "http://planck:3030/$file"
```
Comment (stephenyeargin) [inline modules/autologinpi/boot_into_only_chrome]: Needs a ENV var.

File: modules/autologinpi/boot_into_only_chrome [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# apt-get install -y unclutter
```
Comment (stephenyeargin) [inline modules/autologinpi/boot_into_only_chrome]: Also commented out?

Comment (jameswhite) [inline modules/autologinpi/boot_into_only_chrome]: That's just a note to do it by hand ahead of time.

**Comment by stephenyeargin** [comment]
:shipit:

**Pull request merged**